internal/resources/cluster/cluster_spec_flatten_test.go

```diff
@@ -33,8 +33,9 @@ func TestFlattenSpec(t *testing.T) {
 			},
 			expected: []interface{}{
 				map[string]interface{}{
-					clusterGroupKey: "default",
-					proxyNameKey:    "",
+					clusterGroupKey:      "default",
```

Comment:
This is already taken care of I believe, can you rebase with master?

Reply:
Done!